MetaMask/metamask-extension
# Update Bridge button visibility

## **Description**
* Updates bridge constants with a new supported network (zkSync). This enables the Bridge button in the EthOverview page when the zkSync network is selected
* Updates bridge constants with latest allowlisted tokens. This makes the Bridge button visible in the TokenOverview page when applicable

## **Manual testing steps**
1. Open the extension
2. Select the zkSync network
3. Verify that Bridge button is enabled and opens the portfolio page when clicked
4. Open the extension
5. Select the Ethereum network
6. Import the RETH token: 0xae78736cd615f374d3085123a210448e74fc6393
7. The Bridge button should be visible in the TokenOverview page

## **Screenshots/Recordings**

### **Before**
<img width="362" alt="Screenshot 2023-09-27 at 2 35 18 PM" src="https://github.com/MetaMask/metamask-extension/assets/100321200/ee1c9182-6846-403d-9b2a-6696c2661f9c">
<img width="355" alt="Screenshot 2023-09-27 at 2 35 04 PM" src="https://github.com/MetaMask/metamask-extension/assets/100321200/d8f5da02-b019-4265-8597-33a9e05ff304">



### **After**
<img width="356" alt="Screenshot 2023-09-27 at 2 35 40 PM" src="https://github.com/MetaMask/metamask-extension/assets/100321200/f1af08f6-2b0c-4c9c-aafa-e4eddbc897cf">
<img width="364" alt="Screenshot 2023-09-27 at 2 34 02 PM" src="https://github.com/MetaMask/metamask-extension/assets/100321200/504224b6-11d9-405e-94b6-8171c9c36873">


## **Related issues**

N/A

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
